### DISCUSSION TITLE
Documentation: examples for multi repo structure

### DISCUSSION THREAD

**Discussion by dabcoder** [pulumi/pulumi#17742] [Pulumi in Practice] [answered]
In [this section of the public documentation](https://www.pulumi.com/docs/iac/packages-and-automation/organizing-projects-stacks/#aligning-to-git-repos), it is mentioned that some users prefer to manage a single Pulumi project using more than one repository. 
Whilst there is an example further down below on the same page for [mono repo](https://www.pulumi.com/docs/iac/packages-and-automation/organizing-projects-stacks/#monorepo-with-base-infrastructure-project) environments, it would be good to add another example of a multi repo structure. Perhaps with a use case where a [Pulumi package/component](https://www.pulumi.com/docs/iac/packages-and-automation/pulumi-packages/authoring/) is authored in a specific repository and the configuration of the infrastructure component to create is set in another repo?

**Comment by interurban** [comment, marked as answer]
Thanks David; I've kicked off a docs issue to capture this feedback and suggestion. Let me know if you've got the guidance you need here in the near-term. Here's the issue: https://github.com/pulumi/docs/issues/13350

**Reply by dabcoder** [reply]
Thanks James much appreciated. Happy to provide specific use cases/examples as well. Marking this discussion as resolved for now.
